### PR TITLE
fix: make dashboard consistent

### PR DESCRIPTION
**Pull request by Sboonny**
- [x] I have read [Chapter's contributing guidelines](https://github.com/freeCodeCamp/chapter/blob/main/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update README.md`).
- [x] My pull request targets the `main` branch of Chapter.

This one makes dashboard consistent, but it has issues

### CODE REVIEW

File: client/src/modules/dashboard/Chapters/Users/pages/ChapterUsersPage.tsx [hunk 1]
```diff
@@ -215,24 +215,20 @@ export const ChapterUsersPage: NextPage = () => {
                     {data.chapter ? (
                       <DataTable
                         data={[data.chapter.chapter_users[index]]}
-                        keys={['type', 'values'] as const}
+                        keys={[' ', '_'] as const}
```
Comment (Sboonny) [inline client/src/modules/dashboard/Chapters/Users/pages/ChapterUsersPage.tsx:218]: the underscore is showing, and I am sure it annoys screen readers as well.

**Review by Sboonny — COMMENTED** [review]
Spacing is still unfixable issue

**Comment by gikf** [comment]
`DataTabale` has `showHeader` prop, which can be used to hide header

File: client/src/modules/dashboard/Chapters/Users/pages/ChapterUsersPage.tsx [hunk 2]
```diff
@@ -215,24 +215,25 @@ export const ChapterUsersPage: NextPage = () => {
...
                               align={'flex-start'}
+                              marginBlock={'1em'}
                             >
-                              <Text>Name</Text>
-                              <Text>Email</Text>
-                              <Text>Ops</Text>
-                              <Text>Role</Text>
+                              <Text fontWeight={700}>Name</Text>
+                              <Text fontWeight={700}>Email</Text>
+                              <Text fontWeight={700}>Ops</Text>
```
Comment (ojeytonwilliams) [inline client/src/modules/dashboard/Chapters/Users/pages/ChapterUsersPage.tsx:232]:
```suggestion
                              <Text fontWeight={700}>Actions</Text>
```
To be consistent with the rest.

File: client/src/modules/dashboard/Events/pages/EventsPage.tsx [hunk 3]
```diff
@@ -128,27 +132,31 @@ export const EventsPage: NextPage = () => {
...
-                          <Text>ACTIONS</Text>
+                          {/* todo fix spacing between elements */}
+                          <Text>Status</Text>
+                          <Text>Name</Text>
+                          <Text>Invite only</Text>
+                          <Text>Venue</Text>
+                          <Text>Capacity</Text>
+                          <Text>Streaming url</Text>
+                          <Text>Date</Text>
+                          <Text>Ops</Text>
```
Comment (ojeytonwilliams) [inline client/src/modules/dashboard/Events/pages/EventsPage.tsx:155]:
```suggestion
                          <Text>Actions</Text>
```

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
Thanks @Sboonny, this is coming together.  The only thing that leapt out at me was the inconsistency between ops and action:

File: client/src/modules/dashboard/Sponsors/pages/SponsorsPage.tsx [hunk 4]
```diff
@@ -19,7 +19,9 @@ export const SponsorsPage: NextPage = () => {
         <VStack>
           <Flex w="full" justify="space-between">
             <Heading id="page-heading">Sponsors</Heading>
-            <LinkButton href="/dashboard/sponsors/new">Add new</LinkButton>
+            <LinkButton href="/dashboard/sponsors/new" colorScheme={'blue'}>
```
[the remark this answers is not in the document]
Comment (Sboonny) [reply client/src/modules/dashboard/Sponsors/pages/SponsorsPage.tsx:22]: I am used to if statements to block access, I have opened chapter.freecodecamp.org/dashboard/sponsers, and there were no buttons, so I am worried about nothing like usual.

File: client/src/modules/dashboard/Chapters/Users/pages/ChapterUsersPage.tsx [hunk 5]
```diff
@@ -215,24 +215,25 @@ export const ChapterUsersPage: NextPage = () => {
...
                               align={'flex-start'}
+                              marginBlock={'1em'}
                             >
-                              <Text>Name</Text>
-                              <Text>Email</Text>
-                              <Text>Ops</Text>
-                              <Text>Role</Text>
+                              <Text fontWeight={700}>Name</Text>
+                              <Text fontWeight={700}>Email</Text>
+                              <Text fontWeight={700}>Actions</Text>
```
Comment (Sboonny) [inline client/src/modules/dashboard/Chapters/Users/pages/ChapterUsersPage.tsx:232]:
```suggestion
                              <Text fontWeight={700}>Action</Text>

**Review by ojeytonwilliams — APPROVED** [review]

**Pull request merged**